feature-pack/src/license/full-feature-pack-licenses.xml

```diff
@@ -700,7 +700,7 @@
       <artifactId>javax.persistence-api</artifactId>
```

Comment:
This groupId and artifactId needs to be updated too. Also, when you do that please put this dependency in the new correct alphabetical position, as these elements are kept in alphabetical order by maven GA.

Reply:
thanks, changed full-feature-pack-licenses.xml and moved to just before jakarta.security.enterprise.